src/Dev/FunctionalTest.php

```diff
@@ -160,6 +162,9 @@ protected function withBaseFolder($folder, $callback)
     public function get($url, $session = null, $headers = null, $cookies = null)
     {
         $this->cssParser = null;
+        if (Versioned::get_stage() === Versioned::DRAFT) {
```

Comment:
Versioned isn't a dependency of framework, FYI. :D This will break in some framework-only installations.

Reply:
It is a Dev dependency and the test suite makes direct assumptions about it. So yes, whilst this will break if you don't have versioned installed the tests won't run anyway (as we've found in a number of modules)

Reply:
I've adjusted it slightly so there's less dependence on `Versioned`